README.md

```diff
@@ -41,10 +41,29 @@ Multiple cameras are supported:
  * Basic usb camera support via the [Video for Linux (V4L)](http://linuxtv.org/downloads/v4l-dvb-apis/) drivers
  * Matrix-Vision BlueFox (USB 2.0) and BlueFox3 (USB 3.0) cameras via [mvIMPACT Acquire SDK](http://www.matrix-vision.com/software-drivers-en.html)
  * Basler cameras via the [Pylon Software Suite](https://www.baslerweb.com/en/products/software/basler-pylon-camera-software-suite/)
- * FLIR cameras via the [SPINNAKER SDK](https://www.ptgrey.com/support/downloads)
+ * FLIR cameras via the [SPINNAKER and FLYCAP SDK](https://www.flir.com/support-center/iis/machine-vision/downloads/spinnaker-sdk-flycapture-and-firmware-download/)
```

Comment:
Probably good to point out that Flycap is not compatible with Ubuntu 18.04, and Spinnaker is officially only compatible with only 16.04 and 18.04.

Reply:
Actually, flywap is compatible with 18.04. At least there is a package from April, 4th 2019.

I think the information about the compatibility should rather come from the download side, as it can quickly get outdated.

Reply:
Ah, that's new, thanks for checking! Yes, given that these are changing often, that sounds good.